src/config.ts

```diff
@@ -41,6 +41,17 @@ session, \`$i\` the tab absolute index.`,
     env: 'URL_QUERY',
     arg: 'url-query',
   },
+  customUrlHandler: {
+    doc: `The path to custom page url handler; the following \
```

Comment:
Maybe we need to change the doc comment here, because those variables are actually passed to the exported function.